src/sentry_plugins/splunk/plugin.py

```diff
@@ -259,17 +263,12 @@ def forward_event(self, event, payload):
         if self.host:
             payload["host"] = self.host
 
-        session = http.build_session()
+        client = self.get_client()
+
         try:
             # https://docs.splunk.com/Documentation/Splunk/7.2.3/Data/TroubleshootHTTPEventCollector
-            resp = session.post(
-                self.project_instance,
-                json=payload,
-                # Splunk cloud instances certifcates dont play nicely
-                verify=False,
-                headers={"Authorization": "Splunk {}".format(self.project_token)},
-                timeout=5,
-            )
+            resp = client.request(payload)
+
             if resp.status_code != 200:
```

Comment:
@NisanthanNanthakumar do we need this anymore? Wouldn't we raise an exception if we have an error status code?